goal_src/jak2/engine/gfx/hw/video.gc

```diff
@@ -6,10 +6,18 @@
 ;; dgos: ENGINE, GAME
 
 (define-extern *video-mode* int)
+
 ;; DECOMP BEGINS
 
-(defun set-video-mode ((arg0 symbol))
-  (case arg0
+;; WARN: Return type mismatch int vs none.
+(defun set-video-mode ((tv-format symbol))
+  "Set related settings to the video mode in the settings, [[*video-params*]] and the [[*video-mode*]]
+   `ntsc` has a [[*video-mode*]] value of `0`, where as `pal` has a value of `1`
+
+   Will also set a bunch of common settings related to profiling and the camera to finalize the switch
```

Comment:
and, very annoyingly, if you don't call this function, the camera settings will use the default from jak 1, which makes everything stretched in jak 2 :upside_down_face: